solcjs-get

```diff
@@ -4,10 +4,39 @@
 // as part of the prepublish step.
 
 var pkg = require('./package.json');
-var fs = require('fs');
+var fs = require('fs-extra');
 var https = require('follow-redirects').https;
 var MemoryStream = require('memorystream');
-var keccak256 = require('js-sha3').keccak256;
+var path = require('path');
+var compilerDir = 'bin';
+
+var yargs = require('yargs')
+  .usage('Usage: [options| version]')
```

Comment:
Is this actually supposed to work with a version or only with the whole file name?
Something like `./solcjs-get 0.8.6` or `./solcjs-get 0.8.6` does not not say that the file does not exist. The script tries to download it:
```
Retrieving available version list...
Requested version: v0.8.6
Downloading version v0.8.6
Error downloading file: 404
```